### ISSUE TITLE
[Docs] Update the readme of `manifest-icons` hint

### ISSUE THREAD

**Issue by molant** [webhintio/hint#2351]
> I'm not a big fan of copy pasting from other websites into the readme so we might want to rethink this section entirely before we publish it. Don't think it should be blocking. What do you think?

_Originally posted by @molant in https://github.com/webhintio/hint/diffs_

**Comment by ststimac** [comment]
@molant @antross is this something else that the Docs vendor could work on?

**Comment by antross** [comment]
Yes, I think this would be a great candidate. We've actually held off publishing these hints intending to get the docs updated first.

**Comment by antross** [comment]
Reference to progress on current docs:
* https://github.com/webhintio/hint/blob/master/packages/hint-manifest-icons/README.md
* https://github.com/webhintio/hint/blob/master/packages/hint-manifest-scoped-urls/README.md

**Comment by hxlnt** [comment]
@Reezaali Could we add this update to our docs revision? I can help with this.
